docs/build/build-manually-or-with-triggers.md

```diff
@@ -103,6 +103,17 @@ Make sure that the names of the source branch and the target branch are spelled
 If spaces are used in the name, Appcircle will trim it without spaces.
 :::
 
+:::info
+If you are using Azure DevOps Server or Azure DevOps Services Cloud as a Git provider, the Appcircle build trigger will not run for PR status updates (Approve, Approve with suggestions, Wait for author, Reject) or action changes (Complete, Mark as draft, Abandon).
+
+Appcircle will only run the trigger for PR creation or PR updates.
+
+**PR Status:**
+<Screenshot url='https://cdn.appcircle.io/docs/assets/pr-1.png' />
```

Comment:
@CanKarabulut as of Nov. sprint, we made some changes to our screenshot policy.

In order to make the user understand the interaction location well in the image, from now on we decided to add the full screenshot and add an arrow pointing to the relevant section that's mentioned. Surrounding with a circle or rectangle is also acceptable for pointing the relevant section in the image.

Also, the image names (_file name_) should be more descriptive. Short image names have the possibility of conflict in CDN. There is a chance of overwriting an existing one, which can break other content in the documents.

For this PR, I will remove the images before merging. You can add them later, within another PR. But it's not essential for this case since the "info" text explains the context well.